Sources/Collections/CollectionSkeletonProtocol.swift

```diff
@@ -28,6 +28,6 @@ extension CollectionSkeleton where Self: UIScrollView {
     var estimatedNumberOfRows: Int { return 0 }
     func addDummyDataSource() {}
     func removeDummyDataSource(reloadAfter: Bool) {}
-    func disableUserInteraction() { isScrollEnabled = false }
-    func enableUserInteraction() { isScrollEnabled = true }
+    func disableUserInteraction() { isUserInteractionEnabled = false }
+    func enableUserInteraction() { isUserInteractionEnabled = true }
```

Comment:
The keyboard will appear.

Reply:
if  isScrollEnabled = true   cell will response touch  it will response tableView delegate